### ISSUE TITLE
Do not edit the history please

### ISSUE THREAD

**Issue by l29ah** [waebbl/waebbl-gentoo#112]
It requires users to manually find out what's wrong and re-clone the repo, and the errors involved could be misleading to people who aren't familiar with git.

**Comment by waebbl** [comment]
Could you please be more specific? What do you mean? When did i edit the history?

**Comment by l29ah** [comment]
I have two versions of your repo, one has
```
commit cd7063635647b10788d7bcdefd3dc4cafe3f5e9c
Author: Bernd Waibel <waebbl@gmail.com>
Date:   Thu Jun 13 00:16:35 2019 +0200

    dev-python/pycollada: change python support
    
    drop python-2.7 support
    add python-3.7 support
    
    Package-Manager: Portage-2.3.67, Repoman-2.3.14
    Signed-off-by: Bernd Waibel <waebbl@gmail.com>
```
and the other has this instead of the commit
```
commit ea8174c5643976e4559881c707905095c2ea4b98
Author: Bernd Waibel <waebbl@gmail.com>
Date:   Thu Jun 13 00:16:35 2019 +0200

    dev-python/pycollada: add python-3.7 support
    
    Partly revert commit cd7063635647b10788d7bcdefd3dc4cafe3f5e9c
    due to no revbump needed.
    
    Delete pycollada-0.6-r1.ebuild, don't drop python-2.7 support
    for now.
    
    Package-Manager: Portage-2.3.67, Repoman-2.3.14
    Signed-off-by: Bernd Waibel <waebbl@gmail.com>
```
If you check out your github repo, you will find only the latter commit, while the former one used to be there until you did a force push.

**Comment by waebbl** [comment]
Now I see what you mean.  This was due to a merge conflict I got during rebasing, when I needed to partly revert some changes. Github gave me the hints to solve it the way I did it, which led to a merge with this result.
Usually I try to keep my history linear and clean, but I'm no git guru either and didn't know how to solve this another way as has been told by github. I'm sorry for your inconvenience with this.
